<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="2">
    <item row="1" column="0">
     <widget class="myqgraphicsview" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_choosepicture"/>
   <addaction name="separator"/>
   <addaction name="action_zoomin"/>
   <addaction name="separator"/>
   <addaction name="action_zoomout"/>
   <addaction name="separator"/>
   <addaction name="action_1_to_1"/>
   <addaction name="separator"/>
   <addaction name="action_fitwin"/>
   <addaction name="separator"/>
   <addaction name="action_rect"/>
   <addaction name="separator"/>
   <addaction name="action_rotaterect"/>
   <addaction name="separator"/>
   <addaction name="action_polygon"/>
   <addaction name="separator"/>
   <addaction name="action_circle"/>
   <addaction name="separator"/>
   <addaction name="action_concircle"/>
   <addaction name="separator"/>
   <addaction name="action_ringexpansion"/>
   <addaction name="separator"/>
   <addaction name="action_caliberline"/>
   <addaction name="separator"/>
   <addaction name="action_calibercircle"/>
  </widget>
  <action name="action_choosepicture">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/choosepicture.png</normaloff>:/res/Icon/choosepicture.png</iconset>
   </property>
   <property name="text">
    <string>加载图像</string>
   </property>
   <property name="toolTip">
    <string>加载图像</string>
   </property>
  </action>
  <action name="action_rect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/rect.png</normaloff>:/res/Icon/rect.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="toolTip">
    <string>矩形</string>
   </property>
  </action>
  <action name="action_rotaterect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/rotaterect.png</normaloff>:/res/Icon/rotaterect.png</iconset>
   </property>
   <property name="text">
    <string>旋转矩形</string>
   </property>
   <property name="toolTip">
    <string>旋转矩形</string>
   </property>
  </action>
  <action name="action_polygon">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/polygon.png</normaloff>:/res/Icon/polygon.png</iconset>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
   <property name="toolTip">
    <string>多边形</string>
   </property>
  </action>
  <action name="action_circle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/circle.png</normaloff>:/res/Icon/circle.png</iconset>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
   <property name="toolTip">
    <string>圆</string>
   </property>
  </action>
  <action name="action_concircle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/concircle.png</normaloff>:/res/Icon/concircle.png</iconset>
   </property>
   <property name="text">
    <string>同心圆</string>
   </property>
   <property name="toolTip">
    <string>同心圆</string>
   </property>
  </action>
  <action name="action_ringexpansion">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/ringexpension.png</normaloff>:/res/Icon/ringexpension.png</iconset>
   </property>
   <property name="text">
    <string>圆环展开</string>
   </property>
   <property name="toolTip">
    <string>圆环展开</string>
   </property>
  </action>
  <action name="action_caliberline">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/caliber.png</normaloff>:/res/Icon/caliber.png</iconset>
   </property>
   <property name="text">
    <string>卡尺</string>
   </property>
   <property name="toolTip">
    <string>直线卡尺</string>
   </property>
  </action>
  <action name="action_calibercircle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/caliber_circle.png</normaloff>:/res/Icon/caliber_circle.png</iconset>
   </property>
   <property name="text">
    <string>圆卡尺</string>
   </property>
   <property name="toolTip">
    <string>圆卡尺</string>
   </property>
  </action>
  <action name="action_zoomin">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/zoomin.png</normaloff>:/res/Icon/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>图像放大</string>
   </property>
   <property name="toolTip">
    <string>图像放大</string>
   </property>
  </action>
  <action name="action_zoomout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/zoomout.png</normaloff>:/res/Icon/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>图像缩小 </string>
   </property>
   <property name="toolTip">
    <string>图像缩小 </string>
   </property>
  </action>
  <action name="action_fitwin">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/fitwin.png</normaloff>:/res/Icon/fitwin.png</iconset>
   </property>
   <property name="text">
    <string>自适应窗口</string>
   </property>
  </action>
  <action name="action_1_to_1">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/Icon/1_to_1.png</normaloff>:/res/Icon/1_to_1.png</iconset>
   </property>
   <property name="text">
    <string>图像实际尺寸</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myqgraphicsview</class>
   <extends>QGraphicsView</extends>
   <header>myqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
